docs/apis-tools/camunda-api-rest/specifications/camunda-8-rest-api.info.mdx

```diff
@@ -0,0 +1,60 @@
+---
+id: camunda-8-rest-api
+title: "Camunda 8 REST API"
+description: "API for communicating with the Camunda 8 cluster."
+sidebar_label: Introduction
+sidebar_position: 0
+hide_title: true
+custom_edit_url: null
+---
+
+import ApiLogo from "@theme/ApiLogo";
+import SchemaTabs from "@theme/SchemaTabs";
+import TabItem from "@theme/TabItem";
+import Export from "@theme/ApiExplorer/Export";
+
+<span className={"theme-doc-version-badge badge badge--secondary"}>
+  Version: 0.1
+</span>
```

Comment:
This is the only page where I'm still seeing this version tag.

Reply:
Adjusted 👍